### DISCUSSION TITLE
Support for Rocky/Alma Linux

### DISCUSSION THREAD

**Discussion by DBACHRY13** [simp/simp-core#883] [General]
Wondering when the metadata.json files will be updated for all SIMP modules to allow for support of Rocky a/o Linux?  If CentOS and RedHat are supported, one would think that Rocky/Alma would be as well, deeming they are both downstream like how Centos formerly was.

Is there a general reason why this has not happened yet, or a technical reason?

I have to currently keep copies of a couple of the modules in my puppet control repo instead of leveraging the Puppetfile to just pull them into the environment, just so I can manually update the metadata.json file to add a couple lines so the assertions no longer fail claiming it is not supported.  It would be nice to be able to leverage the Puppetfile download for all of the modules and not have to statically download and link them to environments.

**Comment by silug** [comment]
We held off making the necessary changes while we we working on the 6.6.0 release.  That work is now in progress.
